distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestCatalog.java

```diff
@@ -187,6 +192,43 @@ public void testFilterPlugin() {
         }
     }
 
+    @Test
+    public void testContentDirectoryMonitor() throws Exception {
+        startFeature(true, "content-core-directorymonitor");
+        final String TMP_PREFIX = "tcdm_";
+        Path tmpDir = Files.createTempDirectory(TMP_PREFIX);
+        tmpDir.toFile().deleteOnExit();
+        Path tmpFile = Files.createTempFile(tmpDir, TMP_PREFIX, "_tmp.xml");
+        tmpFile.toFile().deleteOnExit();
+        Files.copy(this.getClass().getClassLoader().getResourceAsStream("metacard5.xml"), tmpFile,
+                StandardCopyOption.REPLACE_EXISTING);
+        LOGGER.debug("RCZ:" + tmpFile.toAbsolutePath().toString());
+
+
+        Map<String, Object> cdmProperties = new HashMap<>();
+        cdmProperties.putAll(getMetatypeDefaults("content-core-directorymonitor",
+                "ddf.content.core.directorymonitor.ContentDirectoryMonitor"));
+        cdmProperties.put("monitoredDirectoryPath", tmpDir.toString() + "/"); // Must end with /
+        cdmProperties.put("directive", "STORE_AND_PROCESS");
+        createManagedService("ddf.content.core.directorymonitor.ContentDirectoryMonitor",
+                cdmProperties);
+
+        long startTime = System.nanoTime();
+        ValidatableResponse response = executeOpenSearch("xml", "q=*SysAdmin*");
+        while (TimeUnit.NANOSECONDS.toSeconds(System.nanoTime() - startTime) < 9) {
```

Comment:
We do have some [time unit constants in AbstractIntegrationTest](https://github.com/codice/ddf/blob/9d031d11552797e1b9a86c05c7f48a7ab7d9fd7c/distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/AbstractIntegrationTest.java#L96-96) that might be better to use here, since it makes it easier to adjust timeouts for tests globally when debugging.